string name                 # name of current instance (empty for meta description)
string type_class_name      # used internally as lookup name (either type_class_name or <type_class, type_class_package> can solve class lookups ambiguosity)
string type_class_package
string type_class
string base_class_package
string base_class
vigir_generic_params/ParameterSetMsg params
